pkg/querier/distributor_queryable.go

```diff
@@ -26,6 +26,13 @@ import (
 	"github.com/grafana/mimir/pkg/util/spanlogger"
 )
 
+type SeriesCardinalityScope string
+
+const (
+	SeriesScopeInMemory SeriesCardinalityScope = "inmemory"
+	SeriesScopeActive   SeriesCardinalityScope = "active"
```

Comment:
Nit: while having the type in the name is arguable, to avoid possible conflicts, I think it's better to start naming with the meaningful part, and then the type-deduplication part, because it's faster for the brain to understand `querier.InMemorySeriesScope` than `querier.SeriesScopeInMemory`.
And I'd drop the `Series` part too, making it just `querier.InMemoryScope`.

---

_Now, if this was my codebase_, I would actually take advantage of golang's package.type naming, and would have a:

```go
package seriesscope

type Scope string

const (
	InMemory Scope = "inmemory"
	Active   Scope = "active"
)
```

So the usages would be: `seriesscope.InMemory`.

However, this seems to go against our usual pattern of _"put as many things as possible in a single package"._